linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/tests/pam_tally2_even_deny_root_absent.fail.sh

```diff
@@ -8,6 +8,18 @@ account required                        pam_tally2.so
 account	required			pam_permit.so
 CAPTC
 
+{{% if product in ["sle12","sle15"] %}}
```

Comment:
The rule is restricted to sle products. So, why the conditionals here? Would be simpler to only update the file.